daemon/src/lib.rs

```diff
@@ -90,7 +90,7 @@ impl<O, W, P> TakerActorSystem<O, W, P>
 where
     O: Handler<oracle::MonitorAttestation>
         + Handler<oracle::GetAnnouncement>
-        + Handler<oracle::Sync>
+        + Handler<oracle::SyncAnnouncements>
```

Comment:
🔧 You can remove this altogether (in a separate commit? :). `oracle::Sync` should be a module-private type, but we made it public to use it in `daemon-tests`. The `TakerActorSystem` doesn't need to know about the internal messages `oracle::Actor` sends to itself (verified locally).